repository-data/application/src/main/resources/hcm-config/namespaces/hee/ctaCard.yaml

```diff
@@ -42,6 +42,15 @@ definitions:
             hipposysedit:primary: false
             hipposysedit:type: Text
             hipposysedit:validators: [optional]
+          /cardImage:
```

Comment:
does the request to support `Card image` for `ctaCard` defined in a separate ticket as I couldn't see it in `NWPS-220` ?